pkg/apis/scorecard/v1alpha2/types.go

```diff
@@ -34,16 +34,18 @@ const (
 
 // ScorecardTestResult contains the results of an individual scorecard test.
 type ScorecardTestResult struct {
-	// Name is the name of the test
+	// Name is the name of the test image
 	Name string `json:"name"`
```

Comment:
Should it not be replaced for `Image string `json:"image"`` too?

Reply:
This makes me think, I should add `Image` field , instead of replacing `Name`. As this struct might be getting used more than couple of places. They might still need `Name` and `Description`.